### DISCUSSION TITLE
What are this conectors for?

### DISCUSSION THREAD

**Discussion by Galeano115** [kachurovskiy/megaels#22] [Q&A] [answered]
I'm currently in the process to make this ELS for my own lathe, but im dealing with the problem of not beeing able to figure out what the Spindle, pulse and Vin+TACHO (especially TACHO, which encoder should i use?) are actually for. I have an idea but not sure if its right. If anyone can help me i will be very thankfull.

**Comment by kachurovskiy** [comment, marked as answer]
Hi @Galeano115, if you can read schematics, look at the bottom row of https://raw.githubusercontent.com/kachurovskiy/megaels/main/PCB_20211205/Schematic_MegaEls_2021-12-05.png - it covers all connector pins.

Alternatively, look on the back of the PCB:

![image](https://user-images.githubusercontent.com/517919/201459649-03d9b773-383e-4bb7-b382-9b0bb409bf9a.png)

SPINDLE is where you connect the rotary encoder that's sending [A/B quadrature pulses](https://en.wikipedia.org/wiki/Incremental_encoder) when the lathe spindle turns.

VIN is where you should supply 5V power for the device.

TACHO a.k.a. T1 is the output pin where you can connect an external RPM display. It's optional to use it (I don't).

PULSE is where you'd connect your [manual pulse generator](https://www.aliexpress.com/item/32949618549.html).

**Reply by Galeano115** [reply]
Thank you, this helped me a lot!
